README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+My name is Mackenzie Whitfield.  
+I moved to Chicago from San Diego two weeks ago. I **love** the beach. My hobbies also include hiking, traveling, reading, and watching movies. Up until now, I have only lived in California.  
+
+# Where I have Lived 
+1. San Diego  
+2. San Francisco  
+3. Los Angeles
+4. Chicago
+
+# Education
```

Comment:
I really like the use of headers, it makes the text really organized. And it looks like a really interesting experience traveling to Greece, looking forward to your photos!